qulice-xml/pom.xml

```diff
@@ -61,6 +61,7 @@
         <dependency>
             <groupId>com.jcabi</groupId>
             <artifactId>jcabi-xml</artifactId>
+            <version>0.17.1</version>
```

Comment:
@carlosmiranda jcabi-xml is also defined in qulice-maven-plugin and the one there will be used in the end. So a dependencyManagement section in the parent pom is needed.